connector/src/main/resources/schema/wildfly-datasources_4_0.xsd

```diff
@@ -1033,8 +1033,9 @@
       <xs:annotation>
         <xs:documentation>
           <![CDATA[[
-            Specifies the name of AS7 module providing this driver.
-            Thios tag is not used in IronJacamar standalone container.
+            Specifies the name of JBoss module providing this driver.
```

Comment:
Sorry, I didn't see this question. It's fine to just update this branch and push it again to your github repo. The pull request will automatically be updated. Best is to add another commit and then user "git rebase -i" and squash the commits into 1.